### PR TITLE
Use team members instead of public org members on About page - separate section for developers and moderators

### PR DESCRIPTION
**Pull request by TheKodeToad**

### CODE REVIEW

File: src/_data/contributors.json [hunk 1]
```diff
@@ -40,10 +40,6 @@
 			"https://avatars.githubusercontent.com/u/5103549?v=4",
 			"https://github.com/jamierocks"
 		],
-		[
-			"https://avatars.githubusercontent.com/u/65276418?v=4",
-			"https://github.com/binex-dsk"
```
Comment (txtsd) [inline src/_data/contributors.json:45]: LOL

**Review by txtsd — APPROVED** [review]
LGTM. Please resolve conflicts though.

**Comment by NotJansel** [comment]
lgtm

File: src/_data/contributors.json (hunk 1 above)
Comment (TheKodeToad) [reply src/_data/contributors.json:45]: They were deleted

**Pull request merged**